cmd/deployments.go

```diff
@@ -41,13 +42,20 @@ var (
 		Name:     "deployments",
 		Category: "Deployment",
 		Usage:    "List app deployments",
-		Flags:    []cli.Flag{&appFlag},
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.IntFlag{Name: "page", Usage: "Page to display", Value: 1},
+			&cli.IntFlag{Name: "per-page", Usage: "Number of deployements to display", Value: 30},
+		},
```

Comment:
It missing the indentation, you should configure your IDE to automatically indent it on saving.